### ISSUE TITLE
Generate derivation path instead of wallet seed

### ISSUE THREAD

**Issue by shawnz** [zealic/vanitygen-hd#7]
Hi there, would it be possible to instead of generating a vanity wallet seed, to generate a vanity derivation path for an existing seed?

**Comment by zealic** [comment]
Yes, I was developed custom derivation path, but need sometime to publish.

**Comment by shawnz** [comment]
Thank you for the notice.
